src/test/java/racingcar/CarGameTest.java

```diff
@@ -0,0 +1,43 @@
+package racingcar;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CarGameTest {
+    @Test
+    void parseCarNameTest(){
+        String text = "hendo,brody,summer";
+        ArrayList<String> nameList = CarGame.parseCarName(text);
+        assertThat(nameList).contains("hendo").contains("brody").contains("summer");
+    }
+
+    @Test
+    void validateCarNameTest(){
+        assertThatThrownBy(() -> {
+            String text = "hendooo,brody,summer";
+            ArrayList<String> nameList = CarGame.parseCarName(text);
+            CarGame.validateCarName(nameList);
+        });
+    }
+
+    @Test
+    void playRoundTest(){
+        ArrayList<Car> carList = new ArrayList<Car>(Arrays.asList(new Car("hendo", 5), new Car("brody", 3)));
```

Comment:
ArrayList의 상위 인터페이스인 List로 바꿔도 될것 같아요.
```suggestion
List<Car> carList = Arrays.asList(new Car("hendo", 5), new Car("brody", 3));

Reply:
네 다형성을 고려하여 지적해주신 부분 외에도 나머지 코드들도 모두 수정하였습니다 !